layouts/default.vue

```diff
@@ -1,5 +1,18 @@
 <template>
-  <div>
-    <Nuxt />
+  <div class="container mx-auto m-0 p-2 h-screen max-w-4xl">
```

Comment:
I noticed that this 4xl sometimes results in not-so-nice layout. Do you think you can find a better way to have a nice default width?

Reply:
the `max-w-4xl` is removed now.